Assets/Scripts/LevelSelect/Camera/VehicleCamera.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DN.Levelselect.CameraMovement
+{
+    /// <summary>
+    /// Here i make the camera move with the car.
```

Comment:
Vermijd jezelf te noemen in de summary, leg gewoon uit wat het script doet. ("This script makes the camera follow the car")